jira/resources.py

```diff
@@ -692,6 +692,15 @@ def update(  # type: ignore[override] # incompatible supertype ignored
 
         super(Issue, self).update(async_=async_, jira=jira, notify=notify, fields=data)
 
+    def get_field(self, field_name: str) -> Any:
+        if field_name.startswith("_"):
+            raise AttributeError(
+                f"A issue field_name cannot start with underscore (_): {item}",
+                field_name,
+            )
+        else:
+            return getattr(self.fields, field_name)
```

Comment:
Please write a docstring for this function and add a comment here that explains why you are using `getattr()`.

My understanding is that you are using `getattr()` so you can get the custom types e.g. `Project` class for the `.project` property

```suggestion
            # use getattr so we are able to return the custom types obtained from `parse_raw()`
            return getattr(self.fields, field_name)
```

```python
    def get_field(self, field_name: str) -> Any:
        """Obtain the (parsed) value from the Issue's field. 

        Args:
            field_name (str): The name of the field to get

        Raises:
            AttributeError: If the field does not exist or if the field starts with an ``_``

        Returns:
            Any: Returns the parsed data stored in the field. For example, "project" would be of class :py:class:`Project`
        """

```

Reply:
Sry, slipped my mind will get it fixed